glide.yaml

```diff
@@ -44,7 +45,7 @@ import:
 - package: github.com/pkg/errors
   version: ^0.8.0
 - package: github.com/jteeuwen/go-bindata
-  version: ~3.0
```

Comment:
There was actually a reason I used ~3.0. I wanted to transparently be able to upgrade up to an 3.1. But anyways, ...